build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -35,9 +35,10 @@ public class PrepareTargets
 
         // All major targets will depend on this in order to ensure variables are set up right if they are run independently
         [Target(
-            nameof(GenerateVersions), 
-            nameof(CheckPrereqs), 
-            nameof(LocateStage0), 
+            nameof(GenerateVersions),
+            nameof(ZipTemplates),
```

Comment:
This should happen during Prepare, not Init.  The difference is that when we run individual targets from our build, they all depend on Init, so they get static things populated.  We don't want to zip these directories every time we run 1 little target.  Instead, this should be on line 21.